internal/network/nsx_manager_subresource.go

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright 2023 VMware, Inc.
+ *    SPDX-License-Identifier: MPL-2.0
+ */
+
+package network
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	validation_utils "github.com/vmware/terraform-provider-vcf/internal/validation"
+	"github.com/vmware/vcf-sdk-go/models"
+)
+
+// NsxManagerNodeSchema this helper function extracts the NSX Manager Node schema, which contains
+// the parameters required to install and configure NSX Manager in a workload domain.
+func NsxManagerNodeSchema() *schema.Resource {
+	return &schema.Resource{
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				Description:  "Name of the NSX Manager virtual machine",
```

Comment:
```suggestion
				Description:  "Name of the NSX Manager appliance",
```

May need an example?